src/runtime/button-api.js

```diff
@@ -174,6 +185,12 @@ export class ButtonApi {
 
     // Add required CSS class, if missing.
     button.classList.add('swg-smart-button');
+    button.addEventListener('click', () =>
+      this.eventManager_.logSwgEvent(
+        AnalyticsEvent.ACTION_SWG_BUTTON_CLICK,
```

Comment:
is this a smart button event? or same as Line 127

Reply:
do you want to distinguish a button click to a smart button than a regular swg button?

Reply:
Both click events will log the same action: ACTION_SWG_BUTTON_CLICK.

The question of distinguishing between SmartButton and regular came up in discussion and we settled on not logging a separate event just for SmartButton clicks since we will be able to distinguish them by looking at the impression that happened before the click anyhow.

Reply:
You don't need a second event but you could pass in an event parameter though. Whatever helps you filter it in the past without having the depend on a previous event.

Reply:
I agree it would be easier to filter out by adding an additional parameter. I believe I brought this up in a conversation with John, Carter and Shams and we landed on not adding any SmartBox-specific parameters though. I can confirm this with John tomorrow when he gets back.

Reply:
All you need to know is if it was a smart box click or not? Actually you can get that from boq-web iframe too if you log it there. Either ways, only useful to have this distinction if it helps you in your analytics.

Reply:
Talked to John this morning and we decided that we shouldn't be adding any additional information into logs and just look at the impression that came before the SmartButton click.